src/setfit/config.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Any, Optional
+
+from transformers import PretrainedConfig
+
+
+@dataclass
+class SetFitModelConfig:
```

Comment:
I think it would be good to align with the `transformers` convention to exclude `Model` from the config name:
```suggestion
class SetFitConfig:
```